lib/game.rb

```diff
@@ -0,0 +1,62 @@
+class Game
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+  attr_accessor :player_1, :player_2, :board
+  attr_reader :winner
+
+  def initialize(player_1 = Human.new('X'), player_2 = Human.new('O'), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2 
+  end
+
+  def over?
+    (@board.full? || won? || draw?) ? true : false
```

Comment:
Won't the board always be full if the game is a draw? I think you could reduce this logically to
`@board.full? || won?`